getstation/catalog
# [ETF-300] DropdownMenu

# DropdownMenu

Validation steps:
- `yarn && yarn start`
- Go to dropdownMenu section
- test all stories and make sure nothing is broken
